SPECS/rpm/pythondeps.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Is this new Mariner code?  Or is this somewhere else..  Looking for correct copyright notice here.